src_app/catalog_app.js

```diff
@@ -108,6 +109,34 @@ class CatalogApp extends React.Component {
     this.setState({ sceneIndex: prevIndex });
   }
 
+  componentDidMount() {
+    if (Platform.OS === 'android') {
```

Comment:
It looks complex. Can we close this implementation in some module and expose a simpler interface for Linking (the same for all platforms)?